cmd/bundle/init.go

```diff
@@ -18,9 +20,52 @@ var gitUrlPrefixes = []string{
 	"git@",
 }
 
-var aliasedTemplates = map[string]string{
-	"mlops-stack":  "https://github.com/databricks/mlops-stacks",
-	"mlops-stacks": "https://github.com/databricks/mlops-stacks",
+type nativeTemplate struct {
+	gitUrl      string
+	description string
+	aliases     []string
+}
+
+var nativeTemplates = map[string]nativeTemplate{
+	"default-python": {
+		description: "The default Python template",
+	},
+	"mlops-stacks": {
+		gitUrl:      "https://github.com/databricks/mlops-stacks",
+		description: "The Databricks MLOps Stacks template. More information can be found at: https://github.com/databricks/mlops-stacks",
```

Comment:
Can we trim this a bit? The line is very long in the help output. Also, folks can trigger initialization to see the same README reference as listed here. @lennartkats-db @arpitjasa-db Thoughts?

Reply:
I'd recommend having the URL in the command help output. The readme is generated after a user enters values for all prompts. 

I have shortened the help string though.